src/open_company_web/components/topic_list.cljs

```diff
@@ -244,25 +374,30 @@
                            fullscreen-force-edit
                            show-add-topic-tip
                            show-second-add-topic-tooltip
-                           show-share-tooltip]}]
+                           show-share-tooltip
+                           show-second-pin-tip
+                           dragging
+                           rerender]}]
     (let [company-slug    (router/current-company-slug)
           company-data    (:company-data data)
-          category-topics (get-category-topics company-data active-topics)
+          company-topics  (vec (map keyword (:sections company-data)))
           card-width      (:card-width data)
           columns-num     (:columns-num data)
           ww              (.-clientWidth (sel1 js/document :body))
           total-width     (case columns-num
                             3 (str (+ (* card-width 3) 40 60) "px")
```

Comment:
some of these constants of: 3, 40, 60, 64, 413 would be better and more clear as defs.